src/jit/importer.cpp

```diff
@@ -14819,6 +14819,11 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 }
                 op1->gtFlags |= GTF_EXCEPT;
 
+                if (prefixFlags & PREFIX_UNALIGNED)
+                {
+                    op1->gtFlags |= GTF_IND_UNALIGNED;
+                }
+
```

Comment:
@BruceForstall I found that ``GenTreeObj`` is derived from ``GenTreeIndir``, and thus used ``GTF_IND_UNALIGNED`` instead of ``GTF_BLK_UNALIGNED`` (which seems to be more generally applicable according to class hierarchy) here.

Are ``GTF_BLK_UNALIGNED`` and ``GTF_IND_UNALIGNED`` mutually exclusive? If not, it seems that we should set both flags for gtNewObjNode path (Please correct me if I am wrong).

BTW, could you let me know if there are any documents about GenTree? I read RuyJIT overview, but the details on each GenTree node seems to be missing.